drivers/npmx/npmx_shell.c

```diff
@@ -361,6 +361,28 @@ static int cmd_charger_charging_current_get(const struct shell *shell, size_t ar
 	return 0;
 }
 
+static bool charger_charging_current_set_helper(const struct shell *shell,
```

Comment:
What do you think about creating one function setting and printing read value, that would take set/get functions as arguments?

Reply:
Bump

Reply:
As we discussed offline, I prefer to stay with the current solution due to the problem of passing pointers to functions that differ in arguments.